server/src/test/java/com/cloud/server/StatsCollectorTest.java

```diff
@@ -511,4 +512,16 @@ public void testPersistVolumeStatsKVM() {
     public void testPersistVolumeStatsVmware() {
         performPersistVolumeStatsTest(Hypervisor.HypervisorType.VMware);
     }
+
+    private Map<String, String> convertJsonToOrderedMap(String json) {
+        Map<String, String> jsonMap = new TreeMap<String, String>();
+        String[] keyValuePairs = json.replace("{", "").replace("}","").split(",");
+        for(String pair: keyValuePairs) {
```

Comment:
give space between `for` and the opening paranthesis `(`

Reply:
Thanks have updated